### ISSUE TITLE
Make Exec optionally verbose

### ISSUE THREAD

**Issue by remogatto** [owenthereal/gotask#17]
It would be nice to make <tt>Exec</tt> helper function optionally verbose in order to output the executed command.

**Comment by owenthereal** [comment]
Good call! I find this useful as well. I've been logging all [executed commands manually](https://github.com/jingweno/gh/blob/master/gh_task.go#L91-L97). This change will greatly improve readability.
